scraper.rb

```diff
@@ -37,11 +37,15 @@ def scrape_list(url)
     }
     data[:name] = 'Kamal Abdel Fattah' if data[:id] == 'akamal' # No name in English version
     data[:source] = URI.join(url, URI.escape(data[:source])).to_s unless data[:source].to_s.empty?
-    ScraperWiki.save_sqlite([:id, :term], data)
+    if data[:name] == 'Vaccant Poste'
+      warn "Member name is '#{data[:name]}'' Skipping writing to database. (Vacated URL #{data[:source]}).".red
```

Comment:
`.red` here will mess up the Morph output, as it doesn't know how to handle these colour codes. I'm not sure the output really buys us anything anyway, so I'd just replace all this with a `next if data[:name] == 'Vaccant Poste'`